### ISSUE TITLE
Fails to load with CCL 1.12-dev.5

### ISSUE THREAD

**Issue by Ambrevar** [mmgeorge/sly-asdf#31]
I get this error:

```
There is no package named "SB-INT" .
   [Condition of type CCL::NO-SUCH-PACKAGE]

Restarts:
 0: [CONTINUE] Retry finding package with name "SB-INT".
 1: [USE-VALUE] Find specified package instead of "SB-INT" .
 2: [MAKE-NICKNAME] Make "SB-INT" be a nickname for package "SLYNK-ASDF".
 3: [RETRY-COMPILE-FILE] Retry compiling #P"/gnu/store/9n45xls8ggjcv9l2aapicnbkibypzmqf-emacs-sly-asdf-0.1.0-3.ad24805/share/emacs/site-lisp/slynk-asdf.lisp"
 4: [SKIP-COMPILE-FILE] Skip compiling #P"/gnu/store/9n45xls8ggjcv9l2aapicnbkibypzmqf-emacs-sly-asdf-0.1.0-3.ad24805/share/emacs/site-lisp/slynk-asdf.lisp"
 5: [RETRY] Retry compiling #<CL-SOURCE-FILE "slynk-asdf" "slynk-asdf">.
```

I guess the easy fix is to add `#+sbcl` before the `breakpoint` defuns.

**Comment by Ambrevar** [comment]
My bad, just realized it recently got fixed in dbf4af05780bb01c1f5f0dc01682888ca0f79fe2.

**Comment by mmgeorge** [comment]
No worries, thanks for the report!
